src/77480.c

```diff
@@ -750,7 +785,68 @@ void render_player(void) {
 
 INCLUDE_ASM(void, "77480", render_player_model);
 
-INCLUDE_ASM(s32, "77480", appendGfx_player);
+void appendGfx_player(void) {
+    PlayerStatus* playerStatus = &gPlayerStatus;
+    Matrix4f sp20, sp60, spA0, spE0;
+    f32 temp_f0;
+    s32 phi_a0;
+
+    temp_f0 = -gCameras[gCurrentCamID].currentYaw;
```

Comment:
Can you declare during initialization or does it have to be after it?

Reply:
declaring during initialization works